namespaces/live-1.cloud-platform.service.justice.gov.uk/hmpps-restricted-patients-api-prod/01-rbac.yaml

```diff
@@ -0,0 +1,45 @@
+---
+kind: RoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: hmpps-restricted-patients-api-prod-admin
+  namespace: hmpps-restricted-patients-api-prod
+subjects:
+  - kind: Group
+    name: "github:dps-tech"
+    apiGroup: rbac.authorization.k8s.io
+  - kind: Group
+    name: "github:dps-production-releases"
+    apiGroup: rbac.authorization.k8s.io
+roleRef:
+  kind: ClusterRole
+  name: admin
+  apiGroup: rbac.authorization.k8s.io
+---
+kind: RoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: hmpps-restricted-patients-api-prod-admin
+  namespace: hmpps-restricted-patients-api-prod
+subjects:
+  - kind: Group
+    name: "github:dps-core"
+    apiGroup: rbac.authorization.k8s.io
+roleRef:
+  kind: ClusterRole
+  name: admin
+  apiGroup: rbac.authorization.k8s.io
+---
+kind: RoleBinding
```

Comment:
Can you remove this role binding in this rbac.yaml file.

We have a new check to look for users raising PR should be in "github:" group, defined in this file.

Reply:
This fixes the pipeline failure https://github.com/ministryofjustice/github-actions/pull/58. The GitHub action was incorrectly returning an error if the namespace isn't found in live or live-1.

Reply:
It shouldn't hold up this PR though as it's not a requirement. If you're happy with the output of the concourse pipeline @vijay-veeranki, please approve.